src/main/java/com/cdancy/bitbucket/rest/features/PullRequestApi.java

```diff
@@ -211,6 +211,32 @@ ActivitiesPage listActivities(@PathParam("project") String project,
                               @Nullable @QueryParam("limit") Integer limit,
                               @Nullable @QueryParam("start") Integer start);
 
+    @Named("pull-request:list-comments")
+    @Documentation({"https://developer.atlassian.com/static/rest/bitbucket-server/latest/bitbucket-rest.html#idm45888278197104"})
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/{project}/repos/{repo}/pull-requests/{pullRequestId}/comments")
+    @Fallback(BitbucketFallbacks.CommentPageOnError.class)
+    @GET
+    CommentPage listComments(@PathParam("project") String project,
+                                  @PathParam("repo") String repo,
+                                  @PathParam("pullRequestId") long pullRequestId,
+                                  @Nullable @QueryParam("limit") Integer limit,
+                                  @Nullable @QueryParam("start") Integer start);
+
+    @Named("pull-request:list-blocker-comments")
+    @Documentation({"https://developer.atlassian.com/server/bitbucket/rest/v815/api-group-pull-requests/#api-api-latest-projects-projectkey-repos-repositoryslug-pull-requests-pullrequestid-blocker-comments-get"})
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/{project}/repos/{repo}/pull-requests/{pullRequestId}/blocker-comments")
+    @Fallback(BitbucketFallbacks.BlockerCommentsPageOnError.class)
+    @GET
+    BlockerCommentsPage listBlockerComments(@PathParam("project") String project,
+                                            @PathParam("repo") String repo,
+                                            @PathParam("pullRequestId") long pullRequestId,
+                                            @Nullable @QueryParam("limit") Integer limit,
+                                            @Nullable @QueryParam("start") Integer start,
+//                                            @Nullable @QueryParam("count") Boolean count,
```

Comment:
@jutoft is this intentional?

Reply:
no it was not, the api is kind of strange in that the input completely changes the output format returned.
so adding count i think changed what was returned.